e2e-tests/k8s-e2e-bootstraping.sh

```diff
@@ -77,11 +77,11 @@ minikube version
 [ -n "$E2E_SETUP_KUBECTL" ] && setup_kubectl
 
 export MINIKUBE_WANTREPORTERRORPROMPT=false
-export MINIKUBE_HOME=$HOME
-export CHANGE_MINIKUBE_NONE_USER=true
-mkdir "$HOME"/.kube || true
-touch "$HOME"/.kube/config
-export KUBECONFIG=$HOME/.kube/config
+#export MINIKUBE_HOME=$HOME
+#export CHANGE_MINIKUBE_NONE_USER=true
+#mkdir "$HOME"/.kube || true
+#touch "$HOME"/.kube/config
+#export KUBECONFIG=$HOME/.kube/config
```

Comment:
Some of this overrides completely broke the environment set up by the `manusa/actions-setup-minikube@v2.0.1`.

Reply:
yes the kubeconfig should be added automatically by the action when creating the minikube environment.